nixos/tests/appliance-repart-image.nix

```diff
@@ -90,9 +90,15 @@ in
     import os
     import subprocess
     import tempfile
+    import json
 
     tmp_disk_image = tempfile.NamedTemporaryFile()
 
+    with open("${nodes.machine.system.build.image}/output.json") as f:
```

Comment:
I don't think this is a super useful tests because it tests internals of the output representation. Overall, I don't think we need a test for the output at all.

Reply:
Works for me, it's all gone.